javalib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -35,8 +36,11 @@ class Throwable protected (s: String, private var e: Throwable,
    */
   private[this] var suppressed: Array[Throwable] = _
 
-  if (writableStackTrace && !LinkingInfo.targetPureWasm)
-    fillInStackTrace()
+
+  linkTimeIf(!LinkingInfo.targetPureWasm) {
+    if (writableStackTrace)
+      fillInStackTrace()
```

Comment:
Consider moving this test inside `fillInStackTrace()` instead. User code can call `fillInStackTrace`; it should link IMO.

Reply:
:+1: https://github.com/scala-wasm/scala-wasm/pull/24/files#diff-fe3cd167d6d9585bc848ec5044211c773e08531b79dc2c815a5a5bcc505862e9R38